metrics-operator/config/samples/metrics_v1alpha3_analysis.yaml

```diff
@@ -16,7 +16,7 @@ spec:
     project: my-project
     stage: dev
     service: svc1
-    foo: bar # can be any key/value pair; NOT only project/stage/service
+    nodename: test # can be any key/value pair; NOT only project/stage/service
```

Comment:
How is "project/stage/service" relevant to the cloud-based Keptn?  It would be useful to give a "for instance" of what sort of information is appropriate here

Reply:
hey are there just to show that if you had in Keptn v1 metrics related to stage project  etc you can store those info and use them like we do for the nodename

Reply:
Got it!